apps/heureka/src/components/Services/Services.tsx

```diff
@@ -26,13 +27,16 @@ export type ServiceType = {
   serviceOwners: string[]
 }
 
-const defaultFilterSettings = {
-  selectedFilters: [],
-  searchTerm: "",
+const initializeFilterSettings = (initialFilters?: InitialFilters): FilterSettings => {
+  const supportGroupFilters =
+    initialFilters?.support_group?.map((sg) => ({ name: "supportGroupCcrn", value: sg })) ?? []
```

Comment:
you're just picking `support_group` whereas `initialFilters` in appProps can have more default filters. I'd suggest to  iterate on the keys of the whole `initialFilters` object.